coding/review.md

```diff
@@ -0,0 +1,14 @@
+# Code Review
+
+## Suggestions for Effective Review
```

Comment:
I don't think we should merge an empty section. Could you remove this heading and the `[tbd]` for now? Anyone can come along later and add this section if they have time.